content/it/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Docker
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  Docker è una technologia software che offre una virtualizzazione a livello del sistema operativo nota come containers.
+
+aka:
+tags:
+- fundamental
+---
+Docker (nelo specifico, Docker Engine) è una technologia software che offre una virtualizzazione a livello del sistema operativo nota come {{< glossary_tooltip text="containers" term_id="container" >}}.
+
+<!--more-->
+
+Docker utilizza delle funzionalità di isolamente del kernel Linux come cgroups e kernele namespaces e un file system union-capable come OverlayFS e altro permettendo a container indipendenti di girare all'interno di una singola istanza Linux, eliminando il sovraccarico nell'avviare e manutenere delle virtual machines (VMs).
```

Comment:
s/kernele/kernel/
I think that we could translate "virtual machines", since "macchine virtuali" is a very common term afaics